build.sbt

```diff
@@ -48,7 +44,6 @@ inThisBuild(Seq(
   homepage := Some(url("https://github.com/gemini-hlsw/gsp-core")),
   addCompilerPlugin("org.typelevel" % "kind-projector" % kindProjectorVersion cross CrossVersion.full),
   resolvers += "Gemini Repository" at "https://github.com/gemini-hlsw/maven-repo/raw/master/releases", // for gemini-locales
-  crossScalaVersions := Seq(scala212Version, scala213Version),
```

Comment:
Wouldn't this publish or 2.13 only?

Reply:
No, this setting comes for free from sbt-gsp.